AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/OnScreenshotAvailable.java

```diff
@@ -31,26 +34,74 @@ public synchronized void onImageAvailable(ImageReader imageReader) {
     }
 
     Image image = imageReader.acquireLatestImage();
-    Bitmap screenshotBitmap = getBitmapFromImage(image);
-    image.close();
-    imageAlreadyProcessed = true;
-    bitmapConsumer.accept(screenshotBitmap);
+    try {
+      Bitmap screenshotBitmap = getBitmapFromImage(image);
+      image.close();
+      imageAlreadyProcessed = true;
+      bitmapConsumer.accept(screenshotBitmap);
+    } catch (ImageFormatException e) {
+      Logger.logError(TAG, "ImageFormatException: " + e.toString());
+      image.close();
```

Comment:
Would the image.close make sense in a finally bock?

Reply:
Sure, that feels a little more clear